pmm-app/tests/verifyAWSRDSMySQLInstance_test.js

```diff
@@ -18,29 +19,39 @@ Scenario('Open Remote Instances Page and verify Discovery and Adding AWS RDS MyS
     remoteInstancesPage.createNewRemoteMySQL();
     pmmInventoryPage.verifyMySQLRemoteServiceIsDisplayed(instanceIdToMonitor);
     await pmmInventoryPage.verifyAgentHasStatusRunning(instanceIdToMonitor);
-});
+  }
+);
 
-Scenario('Open Inventory Page and verify AWS RDS MySQL 5.6 instance has status running @pmm-post-update', async (I, remoteInstancesPage, pmmInventoryPage) => {
+Scenario(
+  'Open Inventory Page and verify AWS RDS MySQL 5.6 instance has status running @pmm-post-update @not-pr-pipeline',
+  async (I, remoteInstancesPage, pmmInventoryPage) => {
     let serviceName = 'rds-mysql56';
     I.amOnPage(pmmInventoryPage.url);
     pmmInventoryPage.verifyMySQLRemoteServiceIsDisplayed(serviceName);
     await pmmInventoryPage.verifyAgentHasStatusRunning(serviceName);
-});
+  }
+);
 
-Scenario('Verify QAN Filters contain AWS RDS MySQL 5.6 after it was added for monitoring', async (I, qanPage, adminPage) => {
+Scenario(
+  'Verify QAN Filters contain AWS RDS MySQL 5.6 after it was added for monitoring @not-pr-pipeline',
+  async (I, qanPage, adminPage) => {
     let environment = 'RDS MySQL 5.6';
     let filter = qanPage.getFilterLocator(environment);
     I.amOnPage(qanPage.url);
     await I.waitForElement(qanPage.fields.iframe, 60);
-    adminPage.applyTimer("5m");
+    adminPage.applyTimer('5m');
     await I.switchTo(qanPage.fields.iframe);
     qanPage.waitForQANPageLoaded();
     I.seeElement(filter);
-});
+  }
+);
 
-xScenario('Verify MySQL Instances Overview Dashboard for AWS RDS MySQL 5.6 data after it was added for monitoring', async (I, remoteInstancesPage, dashboardPage) => {
+xScenario(
+  'Verify MySQL Instances Overview Dashboard for AWS RDS MySQL 5.6 data after it was added for monitoring @not-pr-pipeline',
```

Comment:
This one is very much needed I believe

Reply:
hm, but it was commented already

Reply:
Or you mean whole test suite?

Reply:
I meant this particular file, they should be included because they link to JS changes, basically using remote instance page